.lycheeignore

```diff
@@ -26,6 +26,10 @@ https://geoport.whoi.edu:80/thredds/
 # catch (at least) github userids, of which many in whatsnews, too many --> "too many requests" failures
 https://github.com/[^/]*$
 
+# GitHub seems especially keen to rate-limit "blob" URLs.
+#  (maybe because these are especially associated with abuse?)
+https://github.com/[^/]*/[^/]*/blob/.*
+
```

Comment:
Afraid I can't see any other way around this.

You can see that all the problems in #6459 are rate limits from GitHub, and specifically associated with "blob" URLs - the many other GitHub URLs are all fine.

Slowing lychee right down - no concurrency, 1 thread, slow retries - reduced the number of network errors but never fully eliminated them when I tested locally, even when using the super-powered `scitools-ci` token.